src/mlpack/tests/load_save_test.cpp

```diff
@@ -14,6 +14,10 @@
 #include <mlpack/core.hpp>
 #include <mlpack/core/data/load_arff.hpp>
 #include <mlpack/core/data/map_policies/missing_policy.hpp>
+#include <mlpack/core/data/scaler_methods/minmaxscaler.hpp>
+#include <mlpack/core/data/scaler_methods/maxabsscaler.hpp>
+#include <mlpack/core/data/scaler_methods/standardscaler.hpp>
+#include <mlpack/core/data/scaler_methods/meannormalization.hpp>
```

Comment:
It might be worth adding another test file here, just to keep things from getting too long.  Technically scaling is not loading/saving. :)

Reply:
Added another test file. Have always added test in existing file so was unsure of creating a new one.